src/rpc/methods/eth.rs

```diff
@@ -2792,6 +2792,53 @@ impl RpcMethod<1> for EthGetLogs {
     }
 }
 
+pub enum EthGetFilterLogs {}
+impl RpcMethod<1> for EthGetFilterLogs {
+    const NAME: &'static str = "Filecoin.EthGetFilterLogs";
+    const NAME_ALIAS: Option<&'static str> = Some("eth_getFilterLogs");
+    const N_REQUIRED_PARAMS: usize = 1;
+    const PARAM_NAMES: [&'static str; 1] = ["filter_id"];
```

Comment:
```suggestion
    const PARAM_NAMES: [&'static str; 1] = ["filterId"];
```

Let's have all the parameters in _camelCase_, as outlined in, e.g., https://github.com/ChainSafe/forest/issues/5279. This is not the case for some existing methods that are not part of the common node proposal, but we should strive for consistency.